site/content/entry/yuzu-progress-report-aug-2021/index.md

```diff
@@ -0,0 +1,349 @@
+[(Local Area Network)](https://github.com/yuzu-emu/yuzu/pull/6863) support has been added to all users!
+
+This means that a small selection of games can now be played with a Switch or any other computer running yuzu while connected to the same network (be it wired ethernet, or via 
+WiFi). With a service like ZeroTier (hamachi has not worked so far) or by manually configuring a VPN 
+[(Virtual Private Network)](https://en.wikipedia.org/wiki/Virtual_private_network), this can be extended to global gameplay!
+
+Games with LAN support so far are:
+
+ARMS
+Bayonetta 2
+Duke Nukem 3D: 20th Anniversary World Tour
+Mario & Sonic at the Olympic Games Tokyo 2020
+Mario Golf: Super Rush
+Mario Kart 8 Deluxe
+Mario Tennis Aces
+Pokkén Tournament DX
+Pokémon Sword & Shield (limited game functions by design)
+Saints Row IV®: Re-Elected™
+SAINTS ROW: THE THIRD - THE FULL PACKAGE
+Splatoon 2 (works with auto-stub enabled)
+Titan Quest
+
+Not all the listed games work at the moment due to missing services or just not having the best compatibility right now, but gameplay proved stable in all working cases.
+
+{{< imgs
+	"./lan.png| It just works (Mario Kart 8 Deluxe)"
+  >}}
+
+Keep in mind that some games require certain button combinations to switch between LAN and Local Wireless modes. 
+For example, `Mario Kart 8 Deluxe` needs holding L + R + presing the left analog trigger (L3) in the main menu to be able to create or join a LAN room.
+
+[Morph](https://github.com/Morph1984) later followed with [network interface cleanups](https://github.com/yuzu-emu/yuzu/pull/6905).
+
+## Smooth and glitch-less videos for the win
+
+Thanks to the [introduction of VA-API](https://github.com/yuzu-emu/yuzu/pull/6713) by [yzct12345](https://github.com/yzct12345) back in July, epicboy made it possible to 
+[use hardware video acceleration](https://github.com/yuzu-emu/yuzu/pull/6846) to decode videos with [FFmpeg](https://en.wikipedia.org/wiki/FFmpeg) for different GPU and driver 
+combinations.
+Furthermore, yuzu will gracefully fall back to software decoding in case none of the combinations are supported.
+
+This considerably speeds up the decoding process, improving the performance of the emulator when playing videos.
+
+Currently, the following GPU decoders are implemented:
+
+| GPU | WINDOWS | LINUX |
+| :------: | :------: | :------: |
+|NVIDIA | NVDEC/D3D11VA | NVDEC/VDPAU |
+|AMD | D3D11VA | VA-API/VDPAU |
+|INTEL | D3D11VA | VA-API |
+
+Please note that, since the GPU used to decode videos isn't necessarily the same as the one used for rendering, NVDEC is preferred on Windows over D3D11VA, as the performance 
+experienced with the latter when the iGPU is used for decoding was lower.
+
+Similarly, VA-API is preferred along with the integrated graphics card on Linux (when available), as it was reported that this combination yielded better performance than using 
```

Comment:
This isn't confirmed to be 100% accurate, and yzct never got back to me with further testing of this claim.